mmv1/products/monitoring/api.yaml

```diff
@@ -1084,6 +1084,87 @@ objects:
+    self_link: "v3/{{name}}"
+    update_verb: :PATCH
+    update_mask: true
+    description: |
+      A Service is a discrete, autonomous, and network-accessible unit,
+      designed to solve an individual concern (Wikipedia). In Cloud Monitoring,
+      a Service acts as the root resource under which operational aspects of
+      the service are accessible
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Service Monitoring': 'https://cloud.google.com/monitoring/service-monitoring'
+        'Monitoring API Documentation': 'https://cloud.google.com/monitoring/api/v3/'
+      api: 'https://cloud.google.com/monitoring/api/ref_v3/rest/v3/services'
+    parameters:
+    - !ruby/object:Api::Type::String
+      name: serviceId
+      description: |
+        An optional service ID to use. If not given, the server will generate a
+        service ID.
+      input: true
+    properties:
+    - !ruby/object:Api::Type::String
+      name: name
+      description: |
+        The full resource name for this service. The syntax is:
+        projects/[PROJECT_ID]/services/[SERVICE_ID].
+      output: true
+    - !ruby/object:Api::Type::String
+      name: displayName
+      description: |
+        Name used for UI elements listing this Service.
+    - !ruby/object:Api::Type::KeyValuePairs
+      name: 'userLabels'
+      description: |
+        Labels which have been used to annotate the service. Label keys must start
+        with a letter. Label keys and values may contain lowercase letters,
+        numbers, underscores, and dashes. Label keys and values have a maximum
+        length of 63 characters, and must be less than 128 bytes in size. Up to 64
+        label entries may be stored. For labels which do not have a semantic value,
+        the empty string may be supplied for the label value.
+      send_empty_value: true
+    - !ruby/object:Api::Type::NestedObject
+      name: telemetry
+      description: |
+        Configuration for how to query telemetry on a Service.
+      # Non custom service have non-editable telemetry
+      output: true
+      properties:
+        - !ruby/object:Api::Type::String
+          name: resourceName
+          description: |
+            The full name of the resource that defines this service.
+            Formatted as described in
+            https://cloud.google.com/apis/design/resource_names.
+    - !ruby/object:Api::Type::NestedObject
+      name: basicService
+      description: |
+        A well-known service type, defined by its service type and service labels.
+      # BasicService info can be set on creation but is then immutable.
+      input: true
+      properties:
+        - !ruby/object:Api::Type::String
+          name: serviceType
+          description: |
+            The type of service that this basic service defines, e.g. 
+            APP_ENGINE service type
+          # BasicService info can be set on creation but is then immutable.
+          input: true
```

Comment:
This is implied by basicService having input: true, so can be removed

Reply:
👍